R/makeTreeSummarizedExperimentFromBiom.R

```diff
@@ -151,16 +142,13 @@ makeTreeSEFromBiom <- function(
         })
         # Create a data.frame from the list
         feature_data <- do.call(rbind, feature_data)
-        # Transposing feature_data and make it DFrame object
-        feature_data <- DataFrame(t(feature_data))
+        # Transposing feature_data and make it df object
+        feature_data <- as.data.frame(feature_data)
         # Add correct colnames
         colnames(feature_data) <- colnames
-    # Otherwise if it is already a data.frame clean from artifacts
-    } else if (is(feature_data, "data.frame")) {
-        # Clean feature_data from possible character artifacts
-        feature_data <- DataFrame(.detect_taxa_artifacts_and_clean(feature_data,
-                                                         clean.taxa.names))
     }
+    # Clean feature_data from possible character artifacts
+    feature_data <- .detect_taxa_artifacts_and_clean(feature_data, clean.taxa.names)
```

Comment:
Better to do this in this step --> the input is always data.frame and we save some code lines

Reply:
Ahh you are right you mean moving the detectiong thingy after this [line](https://github.com/microbiome/mia/blob/master/R/makeTreeSummarizedExperimentFromBiom.R#L153) ?

Reply:
or perhaps [here](https://github.com/microbiome/mia/blob/master/R/makeTreeSummarizedExperimentFromBiom.R#L164) and after that cast it to `DataFrame` ?